packs/bestiary-effects/effect-guiding-words.json

```
@@ -4,7 +4,7 @@
     "name": "Effect: Guiding Words",
     "system": {
         "description": {
-            "value": ""
+            "value": "<p>The guide and all allies that can hear or see the guide gain a +1 status bonus to attack and damage rolls against that creature until the beginning of the guide’s next turn.</p>"
```

```suggestion
            "value": "<p>You gain a +1 status bonus to attack and damage rolls against the marked creature.</p>"
```

Also switch this to use TokenMark instead of a toggle